package.json

```diff
@@ -11,17 +11,17 @@
   },
   "main": "index.js",
   "scripts": {
-    "start": "NODE_ENV=development node ./src/server/server.babel.js",
-    "prod": "NODE_ENV=production node ./src/server/server.babel.js",
-    "build": "rimraf build && concurrently \"npm run build:client\" \"npm run build:server\"",
-    "bs": "rimraf build && concurrently \"npm run build:client\" \"npm run build:server\" \"npm run prod\"",
-    "quickstart": "rimraf build && concurrently \"npm run build:db\" \"npm run build:client\" \"npm run build:server\" \"npm run prod\"",
-    "build:client": "NODE_ENV=production webpack --config ./webpack/prod.babel.js",
-    "build:server": "NODE_ENV=production webpack --config ./webpack/server.babel.js",
-    "build:db": "node ./src/server/database/setupDB.babel.js",
-    "test": "NODE_ENV=testing ava ./src/**/__tests___/**/*.js --verbose",
-    "lint": "xo --fix src/**/*.js",
-    "profile": "NODE_ENV=production webpack --config ./webpack/prod.babel.js --profile --json > stats.json"
+    "postinstall":  "concurrently \"npm run build:db\" \"npm run build:client\" \"npm run build:server\" \"npm start\"",
+    "dev":          "NODE_ENV=development node ./src/server/server.babel.js",
+    "start":        "NODE_ENV=production node ./src/server/server.babel.js",
```

Comment:
yeah, totally a user preference issue. My thought is that I dev a lot more than i deploy, and by the time i'm ready to deploy a lot, i'm probably smart enough to rewrite the npm scripts to suit my needs, but i don't have a strong preference either way if there's already an emerging standard